modules/default/calendar/calendar.js

```diff
@@ -133,14 +142,6 @@ Module.register("calendar", {
 			// fetcher till cycle
 			this.addCalendar(calendar.url, calendar.auth, calendarConfig);
 		});
-
```

Comment:
Why did you remove that? Oversight I guess?

Reply:
I didn't find it in the latest commit so I removed it, I'll put it back